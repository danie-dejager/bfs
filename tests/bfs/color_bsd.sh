LSCOLORS="exFxcXDXbxeGxdXb" bfs_diff rainbow -color
