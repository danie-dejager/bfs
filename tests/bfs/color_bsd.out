[34;49m$'rainbow/\e[1m'[0m
[34;49m$'rainbow/\e[1m/'[0m$'\e[0m'
[34;49mrainbow[0m
[34;49mrainbow/[0m[1;33;4;49mpipe[0m
[34;49mrainbow/[0m[1;35;49mbroken[0m
[34;49mrainbow/[0m[1;35;49mchardev_link[0m
[34;49mrainbow/[0m[1;35;49mlink.txt[0m
[34;49mrainbow/[0m[1;39;41msugid[0m
[34;49mrainbow/[0m[1;39;41msuid[0m
[34;49mrainbow/[0m[30;42msticky_ow[0m
[34;49mrainbow/[0m[30;43mow[0m
[34;49mrainbow/[0m[30;46msgid[0m
[34;49mrainbow/[0m[31;49mexec.sh[0m
[34;49mrainbow/[0m[32;4;49msocket[0m
[34;49mrainbow/[0m[37;44msticky[0m
[34;49mrainbow/[0mfile.dat
[34;49mrainbow/[0mfile.txt
[34;49mrainbow/[0mlower.gz
[34;49mrainbow/[0mlower.tar
[34;49mrainbow/[0mlower.tar.gz
[34;49mrainbow/[0mlu.tar.GZ
[34;49mrainbow/[0mmh1
[34;49mrainbow/[0mmh2
[34;49mrainbow/[0mul.TAR.gz
[34;49mrainbow/[0mupper.GZ
[34;49mrainbow/[0mupper.TAR
[34;49mrainbow/[0mupper.TAR.GZ
